src/containers/ipfs/useGetIpfsContentHook.ts.new

```diff
@@ -0,0 +1,169 @@
+import { useState, useEffect } from 'react';
```

Comment:
what different with prev file?

Reply:
this i put by mistake, should be removed